client/app/bundles/shared/components/SideNav.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import css from './SideNav.scss';
+
+
+export default class SideNavBar extends React.Component{
+constructor(props){
+    super(props);
+    this.state = { showMenu: true};
+
+this.handleClick = this.handleClick.bind(this);
+}
+
+handleClick(){
+    this.setState(prevState =>({
+        showMenu: !prevState.showMenu
+    }));
+}
+
+        render(){
+            const containerClass = `${css.container}`;
+            const secondaryListClass = `${css.secondaryList}`;
+            return(
+                <div className={containerClass}>
+                <ul>
+                    <li><a href="#">Dashboard</a></li>
+                    <li onClick={this.handleClick}><a href="#">Moments</a></li>
+                    <div className={secondaryListClass} style = {{display: this.state.showMenu ? 'none' : 'block'}}>
```

Comment:
I used routes in the new version of the sidenav, and when you click Moments it links to the page. I think now it would only show once you're on the moments page. I would have to run some more tests triggering pages to see if this occurs.